vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1187,13 +1195,30 @@ private void processIntentUniversalLink() {
 
     private void onFloatingButtonClick() {
         // ignore any action if there is a pending one
-        if (!isWaitingViewVisible()) {
```

Comment:
Why did you remove this check on isWaitingViewVisible()?
You should keep it before preparing a new mFabDialog